epoxy-compose/build.gradle

```diff
@@ -0,0 +1,60 @@
+plugins {
+  id 'com.android.library'
+  id 'kotlin-android'
+  id 'kotlin-kapt'
+}
+
+android {
+  compileSdkVersion 30
```

Comment:
can you reference sdk targets like the other modules?
```
  compileSdkVersion rootProject.COMPILE_SDK_VERSION

  defaultConfig {
    minSdkVersion rootProject.MIN_SDK_VERSION
    targetSdkVersion rootProject.TARGET_SDK_VERSION
    consumerProguardFiles 'proguard-rules.pro'
  }
  ```
  
  I know we need a higher minsdk here though